packages/lms/src/lib/components/navigation/EntityNav.svelte

```diff
@@ -1,60 +1,112 @@
 <script lang="ts">
-	import type { EntityMeta } from '@mollify/types';
+  import type { EntityMeta } from '@mollify/types';
+  import { readable, writable } from 'svelte/store';
 
-	export let entities: Array<EntityMeta> | EntityMeta = [];
-	$: _entities = Array.isArray(entities) ? entities : [entities];
+  export let entities: Array<EntityMeta> | EntityMeta = [];
+  $: _entities = Array.isArray(entities) ? entities : [entities];
 
-	export let open = false;
+  // Use Svelte writable store for the "open" state
+  let openStates = writable({}); // Initialize as a writable store
+  let isMobile = false; // Initialize the mobile state
 
-	function toggle() {
-		open = !open;
-	}
+  // Check if the code is running in a browser environment
+  if (typeof window !== 'undefined') {
+    // Add a listener to resize events to track the mobile state
+    window.addEventListener('resize', () => {
+      isMobile = window.innerWidth <= 768;
+      if (!isMobile) {
+        isMobile = false; // Close the navigation on wider screens
+      }
+    });
+  }
```

Comment:
I would as they commented above use <svelte:window on:resize={handleResize}> instead. 
It has benefits of cleaning up the event handler and is native to svelte.